utils/api/cases.spec.ts

```diff
@@ -1,62 +1,162 @@
-import axios from 'axios';
-
-import * as casesAPI from './cases';
+import * as relationshipsAPI from './relationships';
```

Comment:
Not sure about these changes, looks like we're changing the `cases` to have `relationships` tests. Probs some Git weirdness?